### PR TITLE
[plugin.video.rtsplaytv] 0.9.2

### PR DESCRIPTION
**Pull request by goggle**
### Description
This is the first release of my new plugin `plugin.video.rtsplaytv`. It is similar to `plugin.video.srfplaytv`, but for the French (instead German) part of Swiss Radio and Television. It currently shares a big amount of code with `plugin.video.srfplaytv`, so I plan to share this code with an additional addon in the future.

So a question for this future pull request: When I outsource that shared code in the future, should I make a big pull request to update `plugin.video.srfplaytv`, `plugin.video.rtsplaytv` and `myaddon.srgtools`, or should I make 3 different pull requests?

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.rtsplaytv/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.vscode
```
Comment (enen92) [inline plugin.video.rtsplaytv/.gitignore]: can you drop this from the submission please?

File: plugin.video.rtsplaytv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <website>https://www.rts.ch/play/tv</website>
+    <email>seileralex@gmail.com</email>
+    <source>https://github.com/goggle/plugin.video.rtsplaytv</source>
+    <news>v0.9.2 (2018-10-07)
+First release
+    </news>
+    <assets>
+        <icon>resources/icon.png</icon>
+        <fanart>resources/fanart.png</fanart>
+        <banner></banner>
```
Comment (enen92) [inline plugin.video.rtsplaytv/addon.xml]: please remove empty tags, if they are not defined just exclude from addon.xml

File: plugin.video.rtsplaytv/resources/lib/rtsplaytv.py [hunk 3]
```diff
@@ -0,0 +1,1377 @@
...
+        list_item.setArt({
+            'thumb': image,
+            'poster': image,
+            'banner': banner,
+        })
+        subs = json_entry.get('subtitleList', [])
+        if subs and SUBTITLES:
+            subtitle_list = [x.get('url') for x in subs if
+                             x.get('format') == 'VTT']
+            if not subtitle_list:
```
Comment (enen92) [inline plugin.video.rtsplaytv/resources/lib/rtsplaytv.py]: you are passing an empty list to setSubtitles if no subtitles are found. Also note vtt is not exactly equivalent to srt, the timestamps may be different than the ones the kodi subrip parser supports.

**Comment by enen92** [comment]
@goggle regarding your question you should make 3 pull requests: 1 per addon. The first one to be merged would, ofc, be the script.module.yourhelperscript

**Comment by Rechi** [comment]
@enen92 this PR contains two commits, so you shouldn't have merged it.

**Comment by enen92** [comment]
Done on the phone, I did not realize there were two commits. My apologies.
Before we were able to squash when merging, what happened to that option?

**Pull request merged**